analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/SymbolicCheck.cs

```diff
@@ -25,6 +25,9 @@ public class SymbolicCheck
         protected SymbolicCheck() { } // Avoid abstract class, fixes S1694
 
         public virtual ProgramState PreProcess(SymbolicContext context) => context.State;
+
```

Comment:
I think we don't need these empty lines in this case. There will be no logic here so it's just a list of methodnames.

Reply:
These are still methods and I don't like exceptions :)